test/dune

```diff
@@ -6,4 +6,4 @@
 
 (env
  (dev
-  (flags (:standard -w -27))))
+  (flags (:standard -w -27-8))))
```

Comment:
To suppress warnings of non-exhaustive pattern matching like `let [a] = create_vars 1 in ...`.